src/cli/require-qunit.js

```diff
@@ -1,3 +1,6 @@
+const path = require('path');
+const { Module } = require('module');
```

Comment:
Any reason for not using strict mode in this & other build/CLI-related files?

Reply:
No. I note that 3 of 5 files in `/src/cli/` have `'use strict';`. And `/bin` and `/build/tasks` have it too. I've fixed this separately in the main branch and rebased this PR. https://github.com/qunitjs/qunit/commit/5b7ab954c002cb9c47fcec095f486c8bf90e95e8